### PR TITLE
Update ADEX info on index.md

### PR DESCRIPTION
**Pull request by cherie-doc**

### CODE REVIEW

File: collections/_technologies/sensor-platforms-and-internet-of-things/sensor-data-exchange/index.md [hunk 1]
```diff
@@ -1,73 +1,53 @@
...
+Agencies can either connect via intranet through the SGWAN interface or via MPLS from their private network. Agencies can use ADEX to discover and share event data such as status updates or event streams in real-time, across internet and intranet. Subscribers will receive updates as soon as the events are published. Publishers can choose who will be able to subscribe.
 
-- Share unclassified (potentially up to restricted) sensor data with government sector organisations
+This diagram shows the components of ADEX and its ability to distribute real-time data across networks. 
 
-An illustration of the components of SDX and its ability to distribute sensor data across networks:
+![How ADEX works](HowADEXWorks.jpg)
 
-![How SDX works](how-sdx-works.png)
+Here is a short introductory video about ADEX.
```
Comment (cherie-doc) [inline collections/_technologies/sensor-platforms-and-internet-of-things/sensor-data-exchange/index.md:25]: [link](https://d3uzjhjcl6zsbz.cloudfront.net/ADEX-overview.mp4)

File: collections/_technologies/sensor-platforms-and-internet-of-things/sensor-data-exchange/index.md [hunk 2]
```diff
@@ -1,73 +1,53 @@
...
+This diagram shows the components of ADEX and its ability to distribute real-time data across networks. 
 
-An illustration of the components of SDX and its ability to distribute sensor data across networks:
+![How ADEX works](HowADEXWorks.jpg)
 
-![How SDX works](how-sdx-works.png)
+Here is a short introductory video about ADEX. 
 
-### Features
+![ADEX Overview](ADEX Overview.mp4)
```
Comment (cherie-doc) [inline collections/_technologies/sensor-platforms-and-internet-of-things/sensor-data-exchange/index.md:27]: Remove this line

File: collections/_technologies/sensor-platforms-and-internet-of-things/sensor-data-exchange/index.md [hunk 3]
```diff
@@ -1,73 +1,53 @@
...
 
-- **Sensor marketplace:** Publish and subscribe to data across government agencies for comprehensive insights
+How to get started on ADEX.
```
Comment (cherie-doc) [inline collections/_technologies/sensor-platforms-and-internet-of-things/sensor-data-exchange/index.md:29]: How to get started on ADEX. Click [here](https://d3uzjhjcl6zsbz.cloudfront.net/how-to-get-started-on-ADEX.mp4)

File: collections/_technologies/sensor-platforms-and-internet-of-things/sensor-data-exchange/index.md [hunk 4]
```diff
@@ -1,73 +1,53 @@
...
 
-- **Support multiple protocols:** MQTT, AMQP, REST, and so on
+![How to get started on ADEX](How to get started on ADEX.mp4)
```
Comment (cherie-doc) [inline collections/_technologies/sensor-platforms-and-internet-of-things/sensor-data-exchange/index.md:31]: Remove this line

File: collections/_technologies/sensor-platforms-and-internet-of-things/sensor-data-exchange/index.md [hunk 5]
```diff
@@ -1,73 +1,53 @@
...
 
-- **Data transfer in real-time:** Enable sensor data to be distributed across government agencies in real-time for timely responses and predictive analytics
+How to set up MQTT connections on ADEX.
```
Comment (cherie-doc) [inline collections/_technologies/sensor-platforms-and-internet-of-things/sensor-data-exchange/index.md:33]: How to set up MQTT connections on ADEX. Click [here](https://d3uzjhjcl6zsbz.cloudfront.net/how-to-set-up-mqtt-connections-on-ADEX.mp4).

File: collections/_technologies/sensor-platforms-and-internet-of-things/sensor-data-exchange/index.md [hunk 6]
```diff
@@ -1,73 +1,53 @@
...
 
-- **Data sharing across network zones:** Secure cross-zone data exchange across internet, intranet, and private networks
+![How to set up MQTT connections on ADEX](How to set up MQTT connections on ADEX.mp4)
```
Comment (cherie-doc) [inline collections/_technologies/sensor-platforms-and-internet-of-things/sensor-data-exchange/index.md:35]: Remove this line

File: collections/_technologies/sensor-platforms-and-internet-of-things/sensor-data-exchange/index.md [hunk 7]
```diff
@@ -1,73 +1,53 @@
...
+ADEX enables government agencies to publish and subscribe to real-time data using a self-service portal that will run in Government Commercial Cloud (GCC) both internet and intranet.
 
-- Publish and subscribe (near) real-time sensor data using a self-service portal both government intranet and government commercial cloud.
+Agencies can either connect via intranet through the SGWAN interface or via MPLS from their private network. Agencies can use ADEX to discover and share event data such as status updates or event streams in real-time, across internet and intranet. Subscribers will receive updates as soon as the events are published. Publishers can choose who will be able to subscribe.
 
-- Share unclassified (potentially up to restricted) sensor data with government sector organisations
+This diagram shows the components of ADEX and its ability to distribute real-time data across networks. 
 
-An illustration of the components of SDX and its ability to distribute sensor data across networks:
+![How ADEX works](HowADEXWorks.jpg)
```
Comment (cherie-doc) [inline collections/_technologies/sensor-platforms-and-internet-of-things/sensor-data-exchange/index.md:23]: ![How ADEX works]image

**Review by weijunyu — APPROVED** [review]

**Pull request merged**